package.json

```diff
@@ -64,7 +64,12 @@
     "postinstall": "npm link --no-audit --no-fund ./.config/plugin/eslint && npm link --no-audit --no-fund eslint-plugin-handsontable && node --experimental-json-modules ./scripts/swap-package-links.mjs",
```

Comment:
Do you think the `postinstall` entry should contain the `install` script for the examples, or the `build` script should trigger it?

Reply:
Examples have their own script for that `npm run examples:install <hot_version>` or maybe I'm missing something. @wojciechczerniak do you have some thoughts?

Reply:
Only if you want to add code snippets as a tests to `npm run test`. Otherwise I would keep it as an addition to the workflow. Until we have all examples tested and they proove their value in the local development process.